go.mod

```diff
@@ -2,13 +2,13 @@ module github.com/ipfs/go-ipfs
 
 require (
 	bazil.org/fuse v0.0.0-20200117225306-7b5117fecadc
-	contrib.go.opencensus.io/exporter/prometheus v0.3.0
+	contrib.go.opencensus.io/exporter/prometheus v0.4.0
 	github.com/blang/semver/v4 v4.0.0
 	github.com/cheggaaa/pb v1.0.29
 	github.com/coreos/go-systemd/v22 v22.3.2
 	github.com/dustin/go-humanize v1.0.0
 	github.com/elgris/jsondiff v0.0.0-20160530203242-765b5c24c302
-	github.com/fsnotify/fsnotify v1.4.9
+	github.com/fsnotify/fsnotify v1.5.1
```

Comment:
https://github.com/fsnotify/fsnotify/releases

While v1.5.0 was a buggy release that got retracted v1.5.1 seems to basically just fix an unsafe pointer conversion (https://github.com/fsnotify/fsnotify/pull/325) and update x/sys

https://github.com/fsnotify/fsnotify/compare/v1.4.9...v1.5.1